judgments/utils/view_helpers.py

```diff
@@ -53,3 +61,31 @@ def get_judgment_by_uri_or_404(uri: str) -> Judgment:
         return get_judgment_by_uri(uri)
     except DocumentNotFoundError:
         raise Http404(f"Judgment not found at {uri}")
+
+
+class DocumentView(TemplateView):
+    def get_context_data(self, **kwargs):
+        document_uri = kwargs["document_uri"]
+
+        document = get_judgment_by_uri_or_404(document_uri)
+
+        context = super().get_context_data(**kwargs)
+        context["document_uri"] = document_uri
+
+        context["document"] = document
+
+        # TODO: Remove this once we fully deprecate 'judgment' contexts
+        context["judgment"] = document
+
+        context["page_title"] = document.name
+        context["courts"] = caselawutils.courts.get_all()
+
+        context.update({"editors": editors_dict()})
```

Comment:
can we simplify this to

```suggestion
        context["editors"] =  editors_dict()
```